arrows/vxl/pixel_feature_extractor.cxx

```diff
@@ -98,18 +118,51 @@ ::check_sub_algorithm( vital::config_block_sptr config, std::string key )
   return true;
 }
 
+// ----------------------------------------------------------------------------
+vil_image_view< vxl_byte >
+pixel_feature_extractor::priv
+::generate_spatial_prior( kwiver::vital::image_container_sptr input_image )
+{
+  auto const ni = static_cast< unsigned >( input_image->width() );
+  auto const nj = static_cast< unsigned >( input_image->height() );
+
+  // Return the previously-computed one if the size is the same
+  if( spatial_prior.ni() == ni && spatial_prior.nj() == nj )
+  {
+    return spatial_prior;
+  }
+
+  spatial_prior = vil_image_view< vxl_byte >( ni, nj, 1 );
+
+  double const scale_factor =
```

Comment:
Minor:
```suggestion
  constexpr double scale_factor =
```
(Probably should have noted that in the previous review; wasn't in my original suggestion because it depends also on `grid_length` being `constexpr`.)